test/integration/hotrestart_test.sh

```diff
@@ -272,8 +283,33 @@ do
   run_testsuite "$HOT_RESTART_JSON" "1" || exit 1
 done
 
+# Hotrestart in specified UDS
+for HOT_RESTART_JSON in "${JSON_TEST_ARRAY[@]}"
```

Comment:
I think it's probably good enough to test the new mode with only 1 of these configs; no need to re-run for all variants.

Reply:
Done.